### PR TITLE
final update on models

### PR DESCRIPTION
**Pull request by mstan19**
models and seed file are updated

### CODE REVIEW

File: server/models/Product.js [hunk 1]
```diff
@@ -20,21 +20,28 @@ const productSchema = new Schema({
     required: true,
     min: 0.99
   },
-  sizes: {
+  gender: {
+    type: String,
+    enum: ["Woman", "Men"],
```
Comment (christylex3) [inline server/models/Product.js:25]: Line 25. "Woman" should be "Women".

File: server/models/User.js [hunk 2]
```diff
@@ -3,6 +3,11 @@ const bcrypt = require("bcrypt");
 const Order = require("./Order");
 
 const userSchema = new Schema({
+    username: {
+        type: String,
+        required: true,
+        trim: true
+    },
```
Comment (christylex3) [inline server/models/User.js:10]: Maybe add a min of 4 chars for username and that it must be unique.

File: server/models/Product.js (hunk 1 above)
Comment (mstan19) [reply server/models/Product.js]: done

File: server/models/User.js (hunk 2 above)
Comment (mstan19) [reply server/models/User.js:12]: done

**Review by christylex3 — COMMENTED** [review]
Sweet. Looks good!

**Review by christylex3 — APPROVED** [review]

**Pull request merged**